src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -72,12 +73,19 @@
          :assets       [rn/flat-list
                         {:render-fn               quo/token-value
                          :data                    temp/tokens
+                         :key                     :assets-list
+                         :content-container-style {:padding-horizontal 8}}]
+         :collectibles [rn/flat-list
+                        {:render-fn               (fn [item]
+                                                    [quo/collectible
+                                                     {:images   [(:image item)]
+                                                      :on-press #(rf/dispatch [:navigate-to
+                                                                               :wallet-collectibles])}])
+                         :data                    temp/collectibles
+                         :key                     :collectibles-list
+                         :key-fn                  :id
+                         :num-columns             2
                          :content-container-style {:padding-horizontal 8}}]
-         :collectibles [quo/empty-state
```

Comment:
curious, I guess we should still keep the empty-state right? as it is something we intent to keep when the user has none. Maybe we pass the mock data from higher up so this code can be left in place? 🤔